### PR TITLE
NaturalNumber

### PR DESCRIPTION
**Pull request by antonredko1993**

### CODE REVIEW

File: ElementaryTasks/Loops.cs [hunk 1]
```diff
@@ -129,13 +129,58 @@ public void Chess()
...
+        }
 
+        public void NaturalNumber()
+        {
+            Console.WriteLine("Enter the value");
+            int.TryParse(Console.ReadLine(), out int userInput);
 
-
-
+            if (userInput <= 1)
```
Comment (GorbunowArtem) [inline ElementaryTasks/Loops.cs]: should be     userInput < 1

File: ElementaryTasks/Loops.cs [hunk 2]
```diff
@@ -129,13 +129,58 @@ public void Chess()
...
+            {
+                Console.WriteLine($"{userInput} isn't correct value");
+            }
+            else
+            {
+                for (int i = 1; i < userInput; i++)
+                {
+                    if (i*i > userInput)
+                    {
+                        Console.Write($"Squuare of number {i}({i*i}) > {userInput}");
```
Comment (GorbunowArtem) [inline ElementaryTasks/Loops.cs]: Square here and below

File: ElementaryTasks/Loops.cs (hunk 1 above)
Comment (antonredko1993) [reply ElementaryTasks/Loops.cs]: done

File: ElementaryTasks/Loops.cs (hunk 2 above)
Comment (antonredko1993) [reply ElementaryTasks/Loops.cs]: done

**Pull request merged**